tests/lifecycle/tests/lifecycle_deployment_scaling.go

```diff
@@ -32,12 +32,14 @@ var _ = Describe("lifecycle-deployment-scaling", func() {
 
 	// 47398
 	It("One deployment, one pod, one container, scale in and out", func() {
+
 		By("Define Deployment")
-		err := globalhelper.CreateAndWaitUntilDeploymentIsReady(lifehelper.DefineDeployment(1, 1, "lifecycleput"),
+		err := globalhelper.CreateAndWaitUntilDeploymentIsReady(
+			lifehelper.DefineDeployment(1, 1, "lifecycleput"),
 			lifeparameters.WaitingTime)
 		Expect(err).ToNot(HaveOccurred())
 
-		By("start lifecycle lifecycle-deployment-scaling")
+		By("Start lifecycle-deployment-scaling")
```

Comment:
Just for consistency with the other tests you might want to add "test" to the end of this, as in:
```suggestion
		By("Start lifecycle-deployment-scaling test")
```